wallet/src/Pos/Wallet/Web/Mode.hs

```diff
@@ -268,3 +334,17 @@ instance (HasConfiguration, HasInfraConfiguration, HasCompileInfo) =>
          MonadTxpLocal WalletWebMode where
     txpNormalize = txNormalize
     txpProcessTx = txProcessTransaction
+
+instance MonadKeys WalletWebMode where
```

Comment:
Copy paste?

Reply:
➕ , better make a polymorphic instance for `HasLens UserSecret` (if compiler allows)

Reply:
>better make a polymorphic instance for HasLens UserSecret (if compiler allows)
it wouldn't allow to defined other instances for `WalletTestMode` (without writing to file)